### ISSUE TITLE
createModeration doesn't exist

### ISSUE THREAD

**Issue by milkmanjr** [openai/openai-node#19]
Can't use the new moderation endpoint with this library and I'd rather use it via the library than making the request myself.

https://beta.openai.com/docs/guides/moderation/overview

**Comment by rob-gordon** [comment]
Leaving this here in case it saves others time until the package has this endpoint

```js
// const key = process.env.OPENAI_API_KEY;

async function moderations(text: string) {
  const res = await fetch("https://api.openai.com/v1/moderations", {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
      Authorization: `Bearer ${key}`,
    },
    body: JSON.stringify({
      input: text,
    }),
  });
  return await res.json();
}
```

**Comment by schnerd** [comment]
Just published `openai@3.0.1` with the moderations endpoint! Sorry this took a while.
